### PR TITLE
Mangata dev v0.9.28

### PR DESCRIPTION
**Pull request by devdanco**

### CODE REVIEW

**Review by mateuszaaa — APPROVED** [review]
lgtm

File: tokens/src/mock.rs [hunk 1]
```diff
@@ -191,20 +192,22 @@ parameter_types! {
...
+	type CandidacyBond = ConstU128<3>;
+	type VotingBondBase = ConstU128<2>;
+	type VotingBondFactor = ConstU128<0>;
 	type LoserCandidate = ();
 	type KickedMember = ();
+	type DesiredMembers = ConstU32<2>;
+	type DesiredRunnersUp = ConstU32<2>;
+	type TermDuration = ConstU64<5>;
+	type MaxCandidates = ConstU32<5>;
+	type MaxVoters = ConstU32<5>;
```
Comment (iStrike7) [inline tokens/src/mock.rs:210]: Why not use the parameters defined above?

File: tokens/src/multi_token_currency.rs [hunk 2]
```diff
@@ -1,239 +1,7 @@
...
-	///
-	/// Returns a signed imbalance and status to indicate if the account was
-	/// successfully updated or update has led to killing of the account.
-	fn make_free_balance_be(
-		currency_id: Self::CurrencyId,
-		who: &AccountId,
-		balance: Self::Balance,
-	) -> SignedImbalance<Self::Balance, Self::PositiveImbalance>;
-}
-
```
Comment (iStrike7) [inline tokens/src/multi_token_currency.rs:236]: Why was this moved to frame support? And why only this trait?

**Review by iStrike7 — COMMENTED** [review]
Where is the Cargo.toml in the root? Was it removed?

**Review by iStrike7 — APPROVED** [review]
